Adding showcase example with namespaces (#227)
epinio/docs: docs/references/namespaces.md

The relevant command is [epinio target](commands/cli/epinio_target.md).

All other namespace management (creation, inspection, deletion) is done through
the [epinio namespace](commands/cli/namespace/epinio_namespace.md) ensemble. 

> A full workflow of namespace can be found in the following [namespace tutorial](docs/tutorials/namespace-tutorial.md).

Reviewer: ```suggestion
The [namespace tutorial](docs/tutorials/namespace-tutorial.md) contains a full set of examples for working with namespaces.
```

Author: Thanks. Addressed in commit c9ce6faa4

The relevant command is [epinio target](commands/cli/epinio_target.md).

All other namespace management (creation, inspection, deletion) is done through
the [epinio namespace](commands/cli/namespace/epinio_namespace.md) ensemble. Following next, there is an example of these commands in action.

## Showcase example with namespaces

Once `Epinio` is deployed, we can check the pre-configured namespace `workspace` using

> `epinio namespace list` :

Author: Thanks. Addressed in commit 20aed9228

```

> ➜  epinio git:(main) epinio namespace list

Author: Thanks. Addressed in commit 20aed9228

> 

Author: Thanks. Addressed in commit 20aed9228

🚢  Listing namespaces

✔️  Epinio Namespaces:
|   NAME    |            CREATED            | APPLICATIONS | CONFIGURATIONS |
|-----------|-------------------------------|--------------|----------------|
| workspace | 2023-03-14 09:48:11 +0100 CET |              |                |
```

> We can create a new namespace using `epinio namespace create NAME`:

Author: Thanks. Addressed in commit 20aed9228

> ```

Author: Thanks. Addressed in commit 20aed9228

➜  epinio git:(main) epinio namespace create newnamespace

🚢  Creating namespace...
Name: newnamespace

✔️  Namespace created.
```

> Also, we can see details of a given namespace like **creation date**, **applications** and **configurations** within using `epinio namespace show NAME`:

Author: Thanks. Addressed in commit 20aed9228

```
➜  epinio git:(main) epinio namespace show newnamespace 

🚢  Showing namespace...
Name: newnamespace

✔️  Details:
|      KEY       |             VALUE             |
|----------------|-------------------------------|
| Name           | newnamespace                  |
| Created        | 2023-03-14 15:27:48 +0100 CET |
| Applications   |                               |
| Configurations |                               |
```

> By default, `Epinio` brings the namespace `workspace` as default. To see this use `epinio target`

Reviewer: @nunix how to best phrase the following things

  - Installing E creates the standard namespace `workspace` on the cluster.
  - After installation, logging into E (i.e. `epinio login`) will set `workspace` as the current/target namespace, if and only if the local `settings.yaml` does not exist. Because only then does the client create the file, with standard contents.
  - If an old `settings.yaml` existed, with some other namespace as current, then the targeted namespace is not changed.

```
➜  epinio git:(main) epinio target   
✔️  
Currently targeted namespace: workspace
```

> Given that we have created earlier a new namespace named `newnamespace` let's target it to operate within:

Reviewer: ```suggestion
Target the namespace `newnamespace` created earlier to operate on its contents:
```

```
➜  epinio git:(main) epinio target newnamespace

🚢  Targeting namespace...
Name: newnamespace

✔️  Namespace targeted.
```

> Finally, let's delete the namespace `workspace` by using `epinio namespace delete NAME`:

Reviewer: ```suggestion
At last delete the namespace `newnamespace` again:
```

Author: Thanks. Addressed in commit 20aed9228
